data/agent_attributes.yaml

```diff
@@ -223,6 +223,12 @@ attributes:
   required: false
   desc: | 
       Directory where the plugins are saved.
+- name: plugins-always-clone-fresh
+  env_var: BUILDKITE_PLUGINS_ALWAYS_CLONE_FRESH
+  default_value: "false"
+  required: false
+  desc: |
+      Force making a fresh clone of plugins every time.  By default, the Buildkite agent will compare a plugin's organisation, repository, and version tag to any plugin checkout it already has, and if they match, simply use that.  Especially during plugin development on a branch, it can be useful to set this option to `true`.
```

Comment:
How about this? 

```suggestion
      Force checking out a fresh clone of plugins every build. By default, if the organisation, repository, and version tag of a plugin checkout match a plugin that is already on the agent, the agent uses that local version. Forcing fresh checkout is especially during plugin development.
```

Reply:
Yes, i think yours is much better!  You just missed one word:

> is especially **[useful]** during plugin development.

Reply:
And is it perhaps clarifying what should match what? (i'm being very nitpicky here so feel free to ignore this one)

> Force checking out a fresh clone of plugins every build. By default, if the organisation, repository, and version tag of a plugin **specification** match a plugin [...]